test/dotcom/system_status/alerts_test.exs

```diff
@@ -0,0 +1,147 @@
+defmodule Dotcom.SystemStatus.AlertsTest do
+  use ExUnit.Case, async: true
+  doctest Dotcom.SystemStatus.Alerts
+
+  import Test.Support.Factories.Alerts.Alert
+
+  alias Dotcom.SystemStatus.Alerts
+
+  describe "active_on_day?/2" do
+    test "returns true if the alert is currently active" do
+      start_time = Timex.now() |> Timex.beginning_of_day()
+      end_time = Timex.now() |> Timex.end_of_day()
+
+      alert = build(:alert, active_period: [{start_time, end_time}])
+
+      time = Faker.DateTime.between(start_time, end_time)
+
+      assert Alerts.active_on_day?(alert, time)
+    end
+
+    test "returns false if the alert starts on the next service day" do
+      start_time = Timex.now() |> Timex.beginning_of_day() |> Timex.shift(hours: 12)
```

Comment:
This doesn't seem to do what the title of the test says? I expect this test to check a particular day to an alert that starts the following day, but it looks this test is kinda doing the opposite?

Reply:
It does, but the way I set it up is pretty opaque (why did I do it this way? Who knows!)

If you look at `time`, it shifts backwards 1 day, so it's acting as if you ran the query "yesterday".

I say "if you look at", but really if you _looked_ at, because this is confusing and I'm going to fix it.